neurolib/utils/functions.py

```diff
@@ -45,6 +45,7 @@ def kuramoto(traces, dt=0.1, smoothing=0.0, peakrange=[0.1, 0.2]):
                 lastMax = m
             phases[n].append(2 * np.pi)
         else:
+            logging.warn("Kuramoto: No peaks found, returning 0.")
```

Comment:
```suggestion
            logging.warning("Kuramoto: No peaks found, returning 0.")
```

as per https://stackoverflow.com/a/15655674

Reply:
Thanks! fixed